v0/destinations/ga4/utils.js

```diff
@@ -57,15 +38,15 @@ function msUnixTimestamp(timestamp) {
   }
 
   if (timeDifferenceInHours <= 0) {
-    if (getDifferenceInMinutes(currentTime, time) > 15) {
+    if (Math.ceil(moment.duration(time.diff(currentTime)).asMinutes()) > 15) {
```

Comment:
Please put some ga4 reference link here for future

Reply:
Didn't find any supporting doc for the future timestamp. Generally, GA4's event builder throws this error when providing a timestamp too far in the future.